website/docs/tutorials/08-cairo-1-support/01-understanding-cairo-packages.md

```diff
@@ -127,5 +127,7 @@ You now use your function with `hello_starknet::mod1::functions::returns_three()
 
 ## Packages and modules names considerations
 
-The names must use only ASCII alphanumeric characters or `_`, and cannot be empty. Additionally, it cannot start with an
-underscore.
\ No newline at end of file
+The name must be a valid Cairo identifier. 
+It must use only ASCII alphanumeric characters or `_`, 
+and cannot be empty, nor can it start with a digit.
+It also must not be a valid Cairo keyword or a wildcard pattern (`_`).
```

Comment:
What do you mean by "wildcard pattern"? If it cannot be an underscore without any other characters I'd write something like 
```suggestion
It also must not be a valid Cairo keyword or a single underscore (`_`).
```